### PR TITLE
Version 2.2.32 and image version 0.0.46

### PR DESCRIPTION
**Pull request by niwilso**
## Change Description

Upgrading from version 2.2.31 to 2.2.32 with minor changes to update dependencies. VERSION-IMAGE-REDACTOR upgrading from 0.0.45 to 0.0.46, now with ability to take kwargs in for ImagePiiVerifyEngine, bug fix, and evaluation code.

## Checklist

- [X] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [ ] I have signed the CLA (if required)
- [X] My code includes unit tests
- [X] All unit tests and lint checks pass locally
- [X] My PR contains documentation updates / additions if required

### CODE REVIEW

**Review by SharonHart — DISMISSED** [review]

**Comment by niwilso** [comment]
Waiting to merge this until the end of the week once we have more PR(s) merged in.

**Review by SharonHart — APPROVED** [review]

**Review by omri374 — APPROVED** [review]

**Pull request merged**
